### PR TITLE
Add more concat op tests with multi inputs.

### PR DESCRIPTION
**Pull request by BruceDai**
@mingmingtasd PTAL, thanks.

### CODE REVIEW

File: test/ops/concat.js [hunk 1]
```diff
@@ -44,7 +83,132 @@ describe('test concat', function() {
...
+        13, 14, 15, 16,
+      ]};
+    testConcatInputs(tensors, expected);
+  });
+
+  it('concat 2d axis=0 with five inputs', function() {
+    const tensors = [
+      {desc: {type: 'float32', dimensions: [1, 2]}, value: [1, 2]},
+      {desc: {type: 'float32', dimensions: [2, 2]}, value: [3, 4, 5, 6]},
+      {
```
Comment (mingmingtasd) [inline test/ops/concat.js:139]: Do you forget formating these lines?

**Review by mingmingtasd — COMMENTED** [review]
LGTM, thanks! @BruceDai

File: test/ops/concat.js (hunk 1 above)
Comment (BruceDai) [reply test/ops/concat.js:139]: Since one line limits 80 characters，@mingmingtasd

Comment (mingmingtasd) [reply test/ops/concat.js:139]: > Since one line limits 80 characters，@mingmingtasd

OK, got it, just a reminder, thanks! Approved yet.@BruceDai

**Review by mingmingtasd — APPROVED** [review]

**Comment by BruceDai** [comment]
Thanks @mingmingtasd, merge it.

**Pull request merged**